pylint/checkers/strings.py

```diff
@@ -750,7 +753,7 @@ def check_for_consistent_string_delimiters(
         Args:
           tokens: The tokens to be checked against for consistent usage.
         """
-        string_delimiters: Counter[str] = collections.Counter()
+        string_delimiters = collections.Counter()  # type: Counter[str]
```

Comment:
Would you mind adding a short comment that this is necessary for `3.6.0`?

Reply:
Good idea, will do. I'll add a comment on the import too.